climpred/metrics.py

```diff
@@ -416,4 +586,4 @@ def _uacc(forecast, reference, dim='svd', comparison=None):
           Forecast Skill. Climate Dynamics, June 9, 2018.
           https://doi.org/10/gd7hfq.
     """
-    return _ppp(forecast, reference, dim=dim, comparison=comparison) ** 0.5
+    return _ppp(forecast, reference, dim=dim, **kwargs) ** 0.5
```

Comment:
Shouldn't this check that **kwargs has comparison in it?

Reply:
the traceback from _ppp with raise the appropriate error